src/main/native/include/sysid/analysis/OLS.h

```diff
@@ -4,6 +4,7 @@
 
 #pragma once
 
+#include <cstddef>
```

Comment:
If you include cstddef, you should be using std::size_t. Doing otherwise is technically undefined behavior (not that the compiler would do anything particularly insane here).